python/example.py

```diff
@@ -11,11 +11,13 @@
 else:
     data = b"some data from python!"
 
-wasm = open("../wasm/code.wasm", 'rb').read()
-hash = hashlib.sha256(wasm).hexdigest()
-config = {"wasm": [{"data": wasm, "hash": hash}], "memory": {"max": 5}}
-
+# a Context provides a scope for plugins to be managed within. creating multiple contexts
+# is expected and groups plugins based on source/tenant/lifetime etc.
 with Context() as context:
+    wasm = open("../wasm/code.wasm", 'rb').read()
+    hash = hashlib.sha256(wasm).hexdigest()
+    config = {"wasm": [{"data": wasm, "hash": hash}], "memory": {"max": 5}}
+
     plugin = context.plugin(config)
     # Call `count_vowels`
     j = json.loads(plugin.call("count_vowels", data))
```

Comment:
@zshipko - to me, it makes more sense to demo the code this way so that it's more apparent that a Context should live longer than a plugin (or span across multiple plugins) vs. a single context along with a plugin. Not saying that was the intent of the code previously, but putting the context creation _after_ the plugin configuration made it appear that the context is unique to a single plugin. 

I could have this wrong though, so please let me know if you think otherwise!